go/flags/endtoend/vtorc.txt

```diff
@@ -59,15 +59,6 @@ Usage of vtorc:
       --topo_global_root string                      the path of the global topology data in the global topology server
       --topo_global_server_address string            the address of the global topology server
       --topo_implementation string                   the topology implementation to use
-      --topo_k8s_context string                      The kubeconfig context to use, overrides the 'current-context' from the config
-      --topo_k8s_kubeconfig string                   Path to a valid kubeconfig file. When running as a k8s pod inside the same cluster you wish to use as the topo, you may omit this and the below arguments, and Vitess is capable of auto-discovering the correct values. https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod
-      --topo_k8s_namespace string                    The kubernetes namespace to use for all objects. Default comes from the context or in-cluster config
-      --topo_zk_auth_file string                     auth to use when connecting to the zk topo server, file contents should be <scheme>:<auth>, e.g., digest:user:pass
-      --topo_zk_base_timeout duration                zk base timeout (see zk.Connect) (default 30s)
-      --topo_zk_max_concurrency int                  maximum number of pending requests to send to a Zookeeper server. (default 64)
-      --topo_zk_tls_ca string                        the server ca to use to validate servers when connecting to the zk topo server
-      --topo_zk_tls_cert string                      the cert to use to connect to the zk topo server, requires topo_zk_tls_key, enables TLS
-      --topo_zk_tls_key string                       the key to use to connect to the zk topo server, enables TLS
```

Comment:
not sure about the other flags, but I'm pretty sure we want to keep these for vtorc.
cc @GuptaManan100

Reply:
Yes, we do, whatever flags we need to connect to a topo-server. We'll need them.

Reply:
I updated the list of  topo binaries in servenv to add `vtorc`. I had just refactored the list from main into this, so I guess previous PRs too had this issue. 

```
func RegisterFlagsForTopoBinaries(registerFlags func(fs *pflag.FlagSet)) {
	topoBinaries := []string{
		"vtbackup",
		"vtcombo",
		"vtctl",
		"vtctld",
		"vtgate",
		"vtgr",
		"vttablet",
		"vttestserver",
		"zk",
		"vtorc",
	}
	for _, cmd := range topoBinaries {
		OnParseFor(cmd, registerFlags)
	}
}
```

Reply:
`zk` doesn't need to be in this list, does it?

Reply:
That list comes from https://github.com/vitessio/vitess/pull/11106/files